maintainers/maintainer-list.nix

```
@@ -15341,6 +15341,12 @@
     githubId = 3958212;
     name = "Tom Sorlie";
   };
+  schinmai-akamai = {
+    email = "schinmai@akamai.com";
+    github = "schinmai-akamai";
+    gitubId = 70169773;
+    name = "Tarun Chinmai Sekar"
```

Seems like you missed a semicolon here, maybe this will fix the `Check that maintainer list is sorted` step:
```suggestion
    name = "Tarun Chinmai Sekar";
```

---

Thanks, fixed now